cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -415,11 +416,15 @@ struct dremel_data {
  * def_level      = { 1, 1, 1,   0,   1, 1}
  * ```
  * @param col Column of LIST type
+ * @param level_nullability Pre-determined nullability at each list level. Empty means infer from
```

Comment:
With this PR, `column_nullable` member of `table_metadata_with_nullability` has a different meaning for ORC and Parquet. This is presumably fine, but needs to be documented as it's currently very easy to misuse the parameter for nested columns (passing a value per column would cause an error). 
We might want to think about overhauling this part of the API to make it easier to use/harder to misuse. What was the feedback from the Spark team?

Reply:
Forgot to even ask. gimme a sec.